src/routes/Ecosystem/MellowLpDepositForm/components/LPMellowVaultDepositInfo.tsx

```diff
@@ -62,27 +66,16 @@ export const LPMellowVaultDepositInfo: React.FunctionComponent<LPMellowVaultDepo
 
       {getCapBar()}
 
-      <Box
-        sx={{
-          display: 'flex',
-          borderRadius: '4px',
-          padding: '8px 16px',
-          border: '1px solid #1E1933',
-          background: '#1E1933',
-          marginTop: '16px',
-        }}
-      >
-        <Typography sx={{ fontSize: '14px', color: '#8B879D' }} variant="h6">
-          YOUR POSITION:
-        </Typography>
-        <Typography sx={{ fontSize: '14px', color: '#4DE5FF', paddingLeft: '8px' }} variant="h6">
+      <PositionBox>
+        <PositionTypography color={'#8B879D'}>YOUR POSITION:</PositionTypography>
```

Comment:
is it possible to get this from constants? colors.....

also usually it is bad idea to pass "style" stuff as props. You can just define 2 different styled components. One for the YOUR POSITION, another for the UserDeposit, and have the colors inside those components (no props)

Reply:
Removed props: https://github.com/Voltz-Protocol/voltz-ui/pull/420/commits/5e3080ec03ce8c7b0dea721fe7ec993a5dfae237

Replaced colors: https://github.com/Voltz-Protocol/voltz-ui/pull/420/commits/d6cedb705445c150c0de0460c9ff447f5ea867d3